references/jurisdictions/fi-prh.md

```
@@ -342,21 +394,54 @@ taxonomy family by accounting framework*).
 
 ## Jurisdiction-specific invariants
 
-### Packaging and artifact shape — XHTML-in-ZIP, NOT `.xbri`
+### Packaging and artifact shape: XHTML-in-ZIP, NOT `.xbri`
 
 **All financial-statement documents must be filed in machine-readable web
 format (XHTML). PDF is not accepted** (Word/PDF may be converted to XHTML
 with free online tools) [S1][S8]. The notification maximum size is **200
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "s8"

(MD052, reference-links-images)